### PR TITLE
[RELEASE START] v3.33.0 > develop

### PR DESCRIPTION
**Pull request by wasp898**
Merge release branch into `develop` after starting release `v3.33.0`

⚠️ Please close PR if changes should not be merged (e.g., support version)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -767,7 +767,7 @@ Version 2.0 of the UI framework is built for player 7.1. If absolutely necessary
 ## 1.0.0 (2017-02-03)
 - First release
 
-[3.32.0]: https://github.com/bitmovin/bitmovin-player-ui/compare/v3.32.0...HEAD
+[3.32.0]: https://github.com/bitmovin/bitmovin-player-ui/compare/v3.32.0...v3.33.0
```
Comment (felix-hoc) [inline CHANGELOG.md:770]:
```suggestion
[3.33.0]: https://github.com/bitmovin/bitmovin-player-ui/compare/v3.32.0...v3.33.0
```

**Review by felix-hoc — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (felix-hoc) [reply CHANGELOG.md]: maybe 🤔

**Review by felix-hoc — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (felix-hoc) [reply CHANGELOG.md]: Problem was that it said `[3.32.0]` in the first place - this should have been `[develop]`

Comment (wasp898) [reply CHANGELOG.md]: But thanks for catching this!!! I kinda forgot this PR was in limbo. 👀

**Pull request merged**